src/screens/ConversationDetails/ConversationDetailsScreen.js

```diff
@@ -31,7 +34,8 @@ const ConversationDetailsScreen = ({ navigation, route }) => {
 
   useEffect(() => {
     dispatch(customAttributeActions.getAllCustomAttributes());
-  }, [dispatch]);
+    fetchConversationWatchers();
```

Comment:
Do you think we can call the conversation watchers action here?

`dispatch(conversationWatchersActions.show({ conversationId }));`